database/level/level.go

```diff
@@ -176,6 +212,12 @@ func New(ctx context.Context, home string, version int) (*Database, error) {
 		return nil, fmt.Errorf("leveldb %v: %w", TransactionsDB, err)
 	}
 
+	// Blocks database is special
+	err = l.openRawDB(BlocksDB, 256*1024*1024)
```

Comment:
Could `256 * 1024 * 1024` be extracted into a constant, so that it is obvious what it is?